pyproject.toml

```
@@ -46,3 +56,7 @@ lint-warnings = "pylint src --exit-zero"
 docs = "make -C doc html"
 docs-clean = "make -C doc clean"
 test-docs = "make -C doc doctest"
+
+[tool.pytest.ini_options]
+testpaths = ['tests/']
+addopts = ['--cov=qibocal', '--cov-report=xml', '--cov-report=html']
```

```suggestion
addopts = ['--cov=qibosoq', '--cov-report=xml', '--cov-report=html']
```
We know that you love qibocal ;-)